src/main/java/org/mqttbee/mqtt/message/publish/mqtt3/Mqtt3PublishView.java

```diff
@@ -97,6 +97,11 @@ public MqttTopic getTopic() {
         return wrapped.getPayload();
     }
 
+    @Override
```

Comment:
@NotNull

Reply:
Probably we should remove @NotNull on methods return primitive types. What do you think?

Reply:
A byte array is not a primitive type and can be null. Everything that can be null has to be annotated with NotNull oder Nullable.

Reply:
DONE